### PR TITLE
Add Java 1.8.0_40+ warnings

### PR DESCRIPTION
**Pull request by Tzky**
* add warning about Mojang shipping 1.8.0_25 with their Windows Launcher
* set required Java version to 1.8.0_40+

### CODE REVIEW

File: source/server/getting-started/jre.rst [hunk 1]
```diff
@@ -4,9 +4,9 @@ Installing Java
 
 Java is needed to run Sponge and Minecraft. You most likely already have Java, but you may need to update it.
 
-Sponge needs Java 8 (or newer) at this time. Older Java versions are deprecated and will not work with Sponge.
-The difference between major versions of Java (6, 7, 8) is fairly significant, and older versions cannot run
-Sponge properly.
+Sponge needs Java 8 (specifically ``1.8.0_40`` or above) at this time. Older Java versions are deprecated and will not
```
Comment (kashike) [inline source/server/getting-started/jre.rst]: needs -> requires

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

File: source/server/getting-started/jre.rst [hunk 2]
```diff
@@ -4,9 +4,9 @@ Installing Java
...
+work with Sponge. The difference between major versions of Java (6, 7, 8) is fairly significant, and older versions
```
Comment (kashike) [inline source/server/getting-started/jre.rst]: Remove "fairly"

File: source/server/getting-started/jre.rst [hunk 3]
```diff
@@ -4,9 +4,9 @@ Installing Java
...
+cannot run Sponge properly.
```
Comment (kashike) [inline source/server/getting-started/jre.rst]: at all, let alone properly

**Comment by Tzky** [comment]
This issue has been resolved in https://github.com/SpongePowered/SpongeForge/pull/487, thus the workaround shown here isn't needed anymore.

**Comment by Tzky** [comment]
That moment when you close the wrong issue. This one is restored for now.

**Comment by Inscrutable** [comment]
It all looks right to me.

**Comment by Tzky** [comment]
Merged in  a118ac5

**Pull request closed without merging**